xml/System.Windows.Forms/Form.xml

```diff
@@ -1149,9 +1149,9 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If the <xref:System.Windows.Forms.Form.ControlBox%2A> property is set to `true`, the control box is displayed in the upper-left corner of the caption bar. The control box is where the user can click to access the system menu.  
+ If the <xref:System.Windows.Forms.Form.ControlBox%2A> property is set to `true`, the control box is displayed in the upper-right corner of the caption bar. The control box can include minimize, maximize, and help buttons in addition to a close button. For the `ControlBox` property to have any effect, you must also set the form's <xref:System.Windows.Forms.Form.FormBorderStyle%2A> property to <xref:System.Windows.Forms.FormBorderStyle.FixedSingle?displayProperty=nameWithType>, <xref:System.Windows.Forms.FormBorderStyle.Sizable?displayProperty=nameWithType>, <xref:System.Windows.Forms.FormBorderStyle.Fixed3D?displayProperty=nameWithType>, or <xref:System.Windows.Forms.FormBorderStyle.FixedDialog?displayProperty=nameWithType>.  
   
- If you set <xref:System.Windows.Forms.Form.ControlBox%2A> to `false`, and also set the <xref:System.Windows.Forms.Form.Location%2A> property, the <xref:System.Windows.Forms.Form.Size%2A> property of Form will not update to reflect that the non-client area of the form has been hidden. To fix this problem, put the code which alters the <xref:System.Windows.Forms.Form.Location%2A> property to the <xref:System.Windows.Forms.Control.HandleCreated> event.  
+ If you set <xref:System.Windows.Forms.Form.ControlBox%2A> to `false` and also set the <xref:System.Windows.Forms.Form.Location%2A> property, the <xref:System.Windows.Forms.Form.Size%2A> property of Form will not update to reflect that the non-client area of the form has been hidden. To fix this problem, put the code which alters the <xref:System.Windows.Forms.Form.Location%2A> property to the <xref:System.Windows.Forms.Control.HandleCreated> event.  
```

Comment:
style Form or lowercase?

will not update -> is not updated?

which alters -> that alters

put the code to the event reads a bit funny. put the code in the event perhaps?

Reply:
Other than removing a stray comma, I hadn't even read this horrible sentence. Thanks, @mairaw. For your last comment, "...in the event handler..."

Reply:
Suggestion:  put the code in the event handler.